client/src/components/ExploreCard.js

```diff
@@ -4,85 +4,82 @@ import { makeStyles } from "@material-ui/core/styles";
-		marginRight: "2rem",
-		height: "2rem",
-		width: "2rem",
-		color: "white",
-	},
-	favoriteIconLiked: {
-		color: `${theme.palette.primary.main}`,
-		marginRight: "2rem",
-		height: "2rem",
-		width: "2rem",
-	},
+  favoriteIcon: {
+    marginRight: "2rem",
+    height: "2rem",
+    width: "2rem",
+    color: "white",
+  },
+  favoriteIconLiked: {
+    color: `${theme.palette.primary.main}`,
+    marginRight: "2rem",
+    height: "2rem",
+    width: "2rem",
+  },
 }));
 
 function ExploreCard(props) {
-	const classes = exploreCardStyles(props);
+  const classes = exploreCardStyles(props);
+  console.log(props);
 
-	const [liked, setLike] = useState(false);
+  const [liked, setLike] = useState(false);
 
-	const handleLiked = () => {
-		setLike((prev) => !prev);
-		// props.onLike();
-	};
-
-	return (
-		<Paper elevation={1} className={classes.mainPaper}>
-			<Grid container direction="column" justify="center">
-				<Grid item xs={12} className={classes.emptyTopSpace}></Grid>
-				<Grid container item xs={12} className={classes.cardDataContainer}>
-					<Grid item xs={6} container justify="flex-start" alignItems="center">
-						<Grid item xs={12}>
-							<p className={classes.locationText}>{props.location},</p>
-						</Grid>
-						<Grid item xs={12}>
-							<p className={classes.locationCountryText}>{props.country}</p>
-						</Grid>
-					</Grid>
-					<Grid item xs={6} container justify="flex-end" alignItems="center">
-						{liked ? (
-							<FavoriteIcon
-								className={classes.favoriteIconLiked}
-								onClick={() => handleLiked()}
-							></FavoriteIcon>
-						) : (
-							<FavoriteIcon
-								className={classes.favoriteIcon}
-								onClick={() => handleLiked()}
-							></FavoriteIcon>
-						)}
-					</Grid>
-				</Grid>
-			</Grid>
-		</Paper>
-	);
+  const handleLiked = () => {
+    setLike((prev) => !prev);
+    // props.onLike();
+  };
+  return (
+    <Paper elevation={1} className={classes.mainPaper}>
+      <Grid container direction="column" justify="center">
+        <Grid item xs={12} className={classes.emptyTopSpace}></Grid>
+        <Grid container item xs={12} className={classes.cardDataContainer}>
+          <Grid item xs={6} container justify="flex-start" alignItems="center">
+            <Grid item xs={12}>
+              <p className={classes.locationText}>{props.location},</p>
+            </Grid>
+            <Grid item xs={12}>
+              <p className={classes.locationCountryText}>{props.country}</p>
+            </Grid>
+          </Grid>
+          <Grid item xs={6} container justify="flex-end" alignItems="center">
+            {liked ? (
+              <FavoriteIcon
+                className={classes.favoriteIconLiked}
+                onClick={() => handleLiked()}
+              ></FavoriteIcon>
+            ) : (
+              <FavoriteIcon
+                className={classes.favoriteIcon}
+                onClick={() => handleLiked()}
+              ></FavoriteIcon>
+            )}
```

Comment:
I know this wasn't your code, but this could be simplified as 
```
<FavoriteIcon
  className={liked ? classes.favoriteIconLiked : classes.favoriteIcon}
  onClick={() => handleLiked()}
></FavoriteIcon>
```